pkg/my/domain.go

```diff
@@ -27,18 +27,18 @@ and returns it as an array of strings.
 func Domains(config config.NetworkInfos, client *node.Client, nickname string) ([]string, error) {
 	const ownedPrefix = "owned"
 
-	wallet, err := wallet.Fetch(nickname)
+	acc, err := wallet.Fetch(nickname)
 	if err != nil {
 		return nil, fmt.Errorf("loading wallet '%s': %w", nickname, err)
```

Comment:
I guess that if we start renaming "wallet" to "account" (where needed), we need to update logs/errors too ? 
```suggestion
		return nil, fmt.Errorf("loading account '%s': %w", nickname, err)
```